__tests__/app.test.js

```diff
@@ -187,4 +187,68 @@ describe("app", () => {
         });
     });
   });
+
+  describe("POST /api/reviews/:review_id/comments", () => {
+    // request body accepts  an object with username and body
+    // responds with POSTed comment
+    test("201: responds with newly created comment", () => {
+      // const requestBody = {
+      //   username: "TheCakeIsALie",
+      //   body: "A fun afternoon! Definitely recommend!!!",
+      // };
+      return request(app)
+        .post("/api/reviews/2/comments")
+        .expect(201)
+        .send({
+          username: "philippaclaire9",
+          body: "A fun afternoon! Definitely recommend!!!"
+        })
+        .then(({ body }) => {
+          expect(body.comment).toMatchObject({
+            comment_id: 7,
+            body: "A fun afternoon! Definitely recommend!!!",
+            votes: 0,
+            author: "philippaclaire9",
+            review_id: 2,
+            created_at: expect.any(String),
+          });
+        });
+    });
+    test("400: GET invalid review_id endpoint when trying to POST comments", () => {
+      return request(app)
+        .post("/api/reviews/cake/comments")
+        .send({
+          username: "philippaclaire9",
+          body: "A fun afternoon! Definitely recommend!!!",
+        })
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.message).toBe("Bad request!");
+        });
+    });
+    test("404: GET responds with error message if trying to POST to a review that doesn't exist but is valid", () => {
+      return request(app)
+        .post("/api/reviews/2000/comments")
+        .send({
+          username: "philippaclaire9",
+          body: "A fun afternoon! Definitely recommend!!!",
+        })
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.message).toBe("Path not found!");
+        });
+    });
+    test("404: GET responds with error message if trying to POST to a review that exist but the data given is not valid", () => {
```

Comment:
Lets change description here to state which data is not valid - ie 404 if  username does not exist